requirements.txt

```diff
@@ -14,7 +14,7 @@ Shapely==1.5.13
 pyproj==1.9.5.1
 pyramid-jwtauth==0.1.3
 bcrypt==2.0.0
-elasticsearch==2.1.0
+elasticsearch==2.2.0
```

Comment:
`2.3.0` is the latest version: https://pypi.python.org/pypi/elasticsearch

Thanks for the pointer.